js/main.js

```diff
@@ -98,7 +103,7 @@ $unl.addEventListener('click', function (event) {
     for (let i = 0; i < data.entries.length; i++) {
       // eslint-disable-next-line eqeqeq
```

Comment:
Don't add eslint rules without asking me! If you're getting an error, it's probably for a reason